lib/projects/__tests__/projects.test.ts

```diff
@@ -1,37 +1,34 @@
-// @ts-nocheck
-const fs = require('fs');
-const os = require('os');
-const path = require('path');
-const { EXIT_CODES } = require('../enums/exitCodes');
-const projects = require('../projects');
-const { logger } = require('@hubspot/local-dev-lib/logger');
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import { EXIT_CODES } from '../../enums/exitCodes';
+import * as projects from '..';
+import { logger } from '@hubspot/local-dev-lib/logger';
 
 jest.mock('@hubspot/local-dev-lib/logger');
 
 describe('lib/projects', () => {
   describe('validateProjectConfig()', () => {
-    let realProcess;
-    let projectDir;
-    let exitMock;
+    let projectDir: string;
+    let exitMock: jest.SpyInstance;
 
     beforeAll(() => {
       projectDir = fs.mkdtempSync(path.join(os.tmpdir(), 'projects-'));
       fs.mkdirSync(path.join(projectDir, 'src'));
-
-      realProcess = process;
     });
 
     beforeEach(() => {
-      exitMock = jest.fn();
-      global.process = { ...realProcess, exit: exitMock };
+      exitMock = jest.spyOn(process, 'exit').mockImplementation((): never => {
+        throw new Error('process.exit() was called.');
```

Comment:
Also, it looks like this is causing the tests to fail

Reply:
I commented above, but it's because it's throwing the error and causing the tests to fail because of it. We can fix it by just returning undefined as never